### ISSUE TITLE
Friendlier error messages when providing invalid script inputs

### ISSUE THREAD

**Issue by josteinaj** [daisy/pipeline-issues#438]
Currently you just get a HTTP 500 error with a stacktrace.

**Comment by capitancambio** [comment]
I get a 400 (BadRequest) when I provide invalid inputs. Do you rememeber ehich type of invalidity did you have?

**Comment by josteinaj** [comment]
You're right, it's a 400:

# 400 : Bad Request
```
&amp;lt;?xml version="1.0" encoding="UTF-8" standalone="no"?&amp;gt;&amp;lt;error query="http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.1" xmlns="http://www.daisy.org/ns/pipeline/data"&amp;gt;&amp;lt;description&amp;gt;Job Request XML is not valid&amp;lt;/description&amp;gt;&amp;lt;trace&amp;gt;java.lang.Throwable: Job Request XML is not valid
	at org.daisy.pipeline.webservice.impl.GenericResource.getErrorRepresentation(GenericResource.java:26)
	at org.daisy.pipeline.webservice.impl.JobsResource.createResource(JobsResource.java:169)
	at sun.reflect.GeneratedMethodAccessor52.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.restlet.resource.ServerResource.doHandle(ServerResource.java:503)
	at org.restlet.resource.ServerResource.post(ServerResource.java:1215)
	at org.restlet.resource.ServerResource.doHandle(ServerResource.java:592)
	at org.restlet.resource.ServerResource.doNegotiatedHandle(ServerResource.java:649)
	at org.restlet.resource.ServerResource.doConditionalHandle(ServerResource.java:348)
	at org.restlet.resource.ServerResource.handle(ServerResource.java:951)
	at org.restlet.resource.Finder.handle(Finder.java:246)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.engine.application.StatusFilter.doHandle(StatusFilter.java:155)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.engine.CompositeHelper.handle(CompositeHelper.java:211)
	at org.restlet.engine.application.ApplicationHelper.handle(ApplicationHelper.java:84)
	at org.restlet.Application.handle(Application.java:381)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Router.doHandle(Router.java:431)
	at org.restlet.routing.Router.handle(Router.java:648)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.engine.application.StatusFilter.doHandle(StatusFilter.java:155)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.routing.Filter.doHandle(Filter.java:159)
	at org.restlet.routing.Filter.handle(Filter.java:206)
	at org.restlet.engine.CompositeHelper.handle(CompositeHelper.java:211)
	at org.restlet.Component.handle(Component.java:392)
	at org.restlet.Server.handle(Server.java:516)
	at org.restlet.engine.ServerHelper.handle(ServerHelper.java:72)
	at org.restlet.engine.adapter.HttpServerHelper.handle(HttpServerHelper.java:152)
	at org.restlet.ext.jetty.JettyServerHelper$WrappedServer.handle(JettyServerHelper.java:170)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:452)
	at org.eclipse.jetty.server.AbstractHttpConnection.content(AbstractHttpConnection.java:894)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.content(AbstractHttpConnection.java:948)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:851)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:77)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:622)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:46)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:603)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:538)
	at java.lang.Thread.run(Thread.java:745)
&amp;lt;/trace&amp;gt;&amp;lt;/error&amp;gt;
```

**Comment by capitancambio** [comment]
Great thanks :D

**Comment by capitancambio** [comment]
https://github.com/daisy/pipeline-framework/pull/80

**Comment by rdeltour** [comment]
Fixed by merging daisy/pipeline-framework#80
